graylog2-web-interface/dev/lintChanges.sh

```diff
@@ -6,4 +6,6 @@
 # It will give you an idea of the ci result, before pushing your changes.
 # You may need to set the correct permissions for this file by running `chmod +x ./dev/lintChanges.sh`
 
-git diff --name-only origin/master...|grep -E '^graylog2-web-interface/(.*).js(x)?$'|sed s,graylog2-web-interface/,,g|xargs yarn lint:path
\ No newline at end of file
+git diff --name-only origin/master...|grep -E '^graylog2-web-interface/(.*).js(x)?$'|sed s,graylog2-web-interface/,,g|xargs yarn lint:path
+
+git diff --name-only origin/master...|grep -E '^graylog2-web-interface/(.*).js(x)?$'|sed s,graylog2-web-interface/,,g|xargs yarn lint:styles
```

Comment:
I guess we'll want to do a similar check in Jenkins and include its results along with ESLint checks. It looks like we need an external formatter for that: [stylelint-checkstyle-formatter](https://github.com/davidtheclark/stylelint-checkstyle-formatter).
The script running in Jenkins is stored in Jenkins itself, but I think it's time to move it to the repo to have versioning and make it harder to change it accidentally. If you feel like it you can take care of that here as well, or just leave it for another PR.